core/controllers/ingest.py

```diff
@@ -48,15 +51,21 @@ def ingest(body, excel_file):
         etl_pipeline = ETL_PIPELINES[source_type]
         workbook = etl_pipeline(workbook)
 
-    schema = current_app.config["VALIDATION_SCHEMA"]
+    schema = (
+        modify_schema(current_app.config["VALIDATION_SCHEMA"])
+        if source_type == "tf_round_one"
+        else current_app.config["VALIDATION_SCHEMA"]
+    )
     cast_to_schema(workbook, schema)
     validation_failures = validate(workbook, schema)
 
     if validation_failures:
         raise ValidationError(validation_failures=validation_failures)
 
     clean_data(workbook)
-    populate_db(workbook, INGEST_MAPPINGS)
+    populate_db_round_one(workbook, INGEST_MAPPINGS) if source_type == "tf_round_one" else populate_db(
+        workbook, INGEST_MAPPINGS
+    )
```

Comment:
Can we maybe use a case match for this? I guess this approach will get quite messy once we have multiple funds/schemas.